java/vector/src/main/java/org/apache/arrow/vector/ipc/ArrowWriter.java

```diff
@@ -98,6 +98,9 @@ public void start() throws IOException {
     ensureStarted();
   }
 
+  /**
+   * Write the vector this vector was constructed with.
```

Comment:
Maybe "Write the record batch currently loaded in the VectorSchemaRoot"?

Reply:
Revised to  "Writes the record batch currently loaded in this instance's VectorSchemaRoot."